packages/clerk-js/src/ui/elements/UserAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Avatar } from '../elements';
+import { PropsOfComponent } from '../styledSystem';
+import { getFullName, getInitials } from '../utils';
+
+type UserAvatarProps = PropsOfComponent<typeof Avatar> & {
+  name?: string | null;
```

Comment:
Where's this `name` used?
Also, let's get `firstName`, `lastName` and `profileImageUrl` with `Pick` from `UserResource`